library/Vanilla/Message.php

```diff
@@ -0,0 +1,76 @@
+<?php
+/**
+ * @copyright 2009-2019 Vanilla Forums Inc.
+ * @license GPL-2.0-only
+ */
+
+namespace Vanilla;
+
+/**
+ * Represent a basic UI message.
+ */
+class Message {
+
+    const TYPE_ALERT = "Alert";
+
+    const TYPE_CASUAL = "Casual";
+
+    const TYPE_INFORMATION = "Info";
+
+    const TYPE_WARNING = "Warning";
+
+    /** @var string */
+    private $body;
+
+    /** @var array */
+    private $classes = ["DismissMessage"];
+
+    /** @var string */
+    private $type;
+
+    /**
+     * Create a new message.
+     *
+     * @param string $body
+     * @param string $type
+     */
+    public function __construct(string $body, string $type) {
```

Comment:
Any way we could reference these as `$htmlBody` or `$unsafeHtmlBody`? At least something indicating that the contents here should be safely processed before being inserted.

I could picture a mistake being made in the future where something one of these is created with some user content reflecting inside of it.

Reply:
I'm not keen on prefixing a required parameter with the word "unsafe", but I agree with making it more apparent that it will be treated as HTML. I think `$htmlBody` sounds good.